packages/gperf.rb

```diff
@@ -3,30 +3,30 @@
 class Gperf < Autotools
   description 'GNU gperf is a perfect hash function generator.'
   homepage 'https://www.gnu.org/software/gperf/'
-  version '3.2-a02b465'
+  version '3.1-97c1fbc'
```

Comment:
We're downgrading?

Reply:
as per the pr description and the commit name, i'm fixing a fake version. version 3.2 does not exist. we're moving to a newer commit hash than the one before, though.

Reply:
Got it.